hwdb.d/60-keyboard.hwdb

```diff
@@ -282,8 +282,8 @@ evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNS50_70MU:*
 evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNV4XMB,ME,MZ:*
 evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNS5x_NS7xPU:*
 evdev:atkbd:dmi:bvn*:bvr*:svnNotebook:pnNV4xPZ:*
- KEYBOARD_KEY_f7=191"
- KEYBOARD_KEY_f8=191"
+ KEYBOARD_KEY_f7=191                                    # Touchpad Toggle
+ KEYBOARD_KEY_f8=191                                    # Touchpad Toggle
```

Comment:
CI fails with the following:
```
/home/runner/work/systemd/systemd/build/../hwdb.d/60-keyboard.hwdb: 221 match groups, 327 matches, 1038 properties
Keycode KEY_191 unknown
Keycode KEY_191 unknown
```

Reply:
Is this `f21`? From linux/input-event-codes.h, 
```c
#define KEY_F21                  191
```

Reply:
Hey @yuwata, sorry for the late reply. That's right, Clevo EC firmware generates identically codes like System76: https://github.com/systemd/systemd/blob/main/hwdb.d/60-keyboard.hwdb#L1816. Hope it will pass now